src/index.ts

```diff
@@ -48,6 +48,16 @@ io.on("connection", socket => {
     }
   );
 
+  socket.on(
+    "server-volatile-broadcast",
+    (roomID: string, encryptedData: ArrayBuffer, iv: Uint8Array) => {
+      console.log(`${socket.id} sends volatile update to ${roomID}`);
```

Comment:
Do you want to switch to a logging framework like `debug` so you don’t spend time printing and storing these in production or is it helpful for debugging issues on prod?

Reply:
yea, makes sense :) I'll make the switch.